### PR TITLE
Update helm charts

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

File: helm-charts/prometheus_values.yaml [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+  alertmanager:
+    create: false
+
 server:
   ingress:
     enabled: true
     annotations:
       kubernetes.io/ingress.class: private-nginx
+    hosts:
+      - kofw9hx1qbrq9dj8nuuwiyug6h.prometheus.internal.test.cloud.mattermost.com
```
Comment (gabrieljackson) [inline helm-charts/prometheus_values.yaml]: Should this be hard-coded for every k8s cluster?

Comment (stylianosrigas) [reply helm-charts/prometheus_values.yaml]: Nope this can be removed. Its automatically added by the Provisioner.

Comment (cpanato) [reply helm-charts/prometheus_values.yaml]: forgot
will update :(
thanks for the catch

Comment (cpanato) [reply helm-charts/prometheus_values.yaml]: done @gabrieljackson @stylianosrigas

**Pull request merged**